pkg/localcontroller/db/db.go

```diff
@@ -71,12 +71,26 @@ func SaveResource(name string, typeMeta, objectMeta, spec interface{}) error {
 			klog.Errorf("failed to update resource(name=%s): %v", name, err)
 			return err
 		}
-		klog.V(2).Infof("updated resource(name=%s)", name)
+		klog.Infof("updated resource(name=%s)", name)
 	}
 
 	return nil
 }
 
+// GetResource gets resource info in db
+func GetResource(name string) *Resource {
+	r := Resource{}
+
+	queryResult := dbClient.Where("name = ?", name).First(&r)
+	if queryResult.RowsAffected == 0 {
+		klog.Warningf("resource(name=%s) not in db", name)
+		return nil
+	}
+
+	klog.Infof("get resource(name=%s)", name)
```

Comment:
why we use info level, instead of debug? I think it's enough set debug level

Reply:
have been deleted it